### PR TITLE
PRM Automated Test - LPS-192014 | master

### PR DESCRIPTION
**Pull request by marinaasant0s**
[LPS-192014](https://liferay.atlassian.net/browse/LPS-192014) - Automate PRMMDFRequestActivities#CanSubmitMDFWithContentMarketingActivity

### CODE REVIEW

**Comment by marinaasant0s** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 1]
```diff
@@ -550,6 +550,28 @@ definition {
 				value1 = "Liferay Brand");
 		}
 
+		if ((${typeOfActivity} == "Content Marketing") && ((${tactic} == "Blog or Article") || (${tactic} == "Business White Paper or eBook") || (${tactic} == "Infographic") || (${tactic} == "Social Content")|| (${tactic} == "Success Story Case Study")|| (${tactic} == "Technical White Paper")|| (${tactic} == "Video")|| (${tactic} == "Web content")|| (${tactic} == "Other"))) {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro]: Hi @marinaasant0s, as all the fields are the same to all these tactics, you don't need to put all these conditions. Your logic could be just for the typeOfActivity.

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by AfonsoHSantos** [comment]
Hi @marinaasant0s, to avoid [this type of commit](https://github.com/pt-liferay-solutions/liferay-portal/pull/4547/commits/eac428cdd3419f71f751e29ba17b1e92ae9cb598), please remember to run `ant format-source-local-changes` before start your commits.

**Comment by marinaasant0s** [comment]
ci:test:sf

**Comment by marinaasant0s** [comment]
Resent #4572

**Pull request closed without merging**